### PR TITLE
[DSD-5159] Added deployment scripts for mosipcertmanager

### PR DESCRIPTION
**Pull request by VSIVAKALYAN**

### CODE REVIEW

File: deployment/v3/mosip/mosipcertmanager/install.sh [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# Installs sample print service
+## Usage: ./restart.sh [kubeconfig]
+
+if [ $# -ge 1 ] ; then
+  export KUBECONFIG=$1
+fi
+
+
+NS=mosipcertmanager
+CHART_VERSION=12.0.2
```
Comment (syedsalman3753) [inline deployment/v3/mosip/mosipcertmanager/install.sh:11]: please change the version to 0.0.1-develop

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/v3/mosip/mosipcertmanager/install.sh (hunk 1 above)
Comment (VSIVAKALYAN) [reply deployment/v3/mosip/mosipcertmanager/install.sh]: Updated Chart version hence closing this Conversation

File: deployment/v3/mosip/mosipcertmanager/install.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+kubectl create ns $NS 
+
+function installing_mosipcertmanager() {
+  echo Istio label
+  kubectl label ns $NS istio-injection=disabled --overwrite
+  helm repo update
+
+  echo Copy configmaps
+  sed -i 's/\r$//' copy_cm.sh
+  kubectl -n $NS delete --ignore-not-found=true cm s3
```
Comment (syedsalman3753) [inline deployment/v3/mosip/mosipcertmanager/install.sh:23]: Please remove this line 23

File: deployment/v3/mosip/mosipcertmanager/install.sh [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  S3_USER_KEY=$( kubectl -n s3 get cm s3 -o json  |jq -r '.data."s3-user-key"' )
+  S3_REGION=$( kubectl -n s3 get cm s3 -o json  |jq -r '.data."s3-region"' )
+
+  read  -p "Please provide DB host name ( Default: postgres-postgresql.postgres ) : " DB_HOST
+  DB_HOST=${DB_HOST:-$DEFAULT_DB_HOST}
+
+  if [ -z $DB_HOST ]; then
+    echo "Host name not provided; EXITING;"
+    exit 1;
+  fi
```
Comment (syedsalman3753) [inline deployment/v3/mosip/mosipcertmanager/install.sh:40]: please remove this

File: deployment/v3/mosip/mosipcertmanager/install.sh [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+
+
+  echo Installing mosipcertmanager
+  helm -n $NS install mosipcertmanager mosip/mosipcertmanager --wait --version $CHART_VERSION \
+  --set mosipcertmanager.configmaps.db.db-server="$DB_HOST" \
+  --set mosipcertmanager.configmaps.s3.s3-region="$S3_REGION" \
+  --set mosipcertmanager.configmaps.s3.s3-host='minio.minio:9000' \
+  --set mosipcertmanager.configmaps.s3.s3-user-key="$S3_USER_KEY"
```
Comment (syedsalman3753) [inline deployment/v3/mosip/mosipcertmanager/install.sh:49]: remove from 45 starting from / till line 49

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
README file missing

File: deployment/v3/mosip/mosipcertmanager/install.sh (hunk 2 above)
Comment (VSIVAKALYAN) [reply deployment/v3/mosip/mosipcertmanager/install.sh]: Taken Care & hence resolving Conversation.

File: deployment/v3/mosip/mosipcertmanager/install.sh (hunk 3 above)
Comment (VSIVAKALYAN) [reply deployment/v3/mosip/mosipcertmanager/install.sh]: Taken Care & hence resolving Conversation.

**Comment by VSIVAKALYAN** [comment]
Added README.md

File: deployment/v3/mosip/mosipcertmanager/install.sh (hunk 4 above)
Comment (VSIVAKALYAN) [reply deployment/v3/mosip/mosipcertmanager/install.sh]: Taken Care & hence resolving Conversation.

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**